Tests/CarthageKitTests/NetrcSpec.swift

```diff
@@ -0,0 +1,180 @@
+                expect(machine?.password) == "qwerty"
+            }
+            
+            it("should load machines for a given multi-line + whitespaces format") {
+                let content = """
+                    machine  example.com login     anonymous
+                    password                  qwerty
+                    """
+                
+                let machines = Netrc.from(content).value?.machines
+                expect(machines?.count) == 1
+                
+                let machine = machines?.first
+                expect(machine?.name) == "example.com"
+                expect(machine?.login) == "anonymous"
+                expect(machine?.password) == "qwerty"
+            }
+            
+            it("should load multiple machines for a given inline format") {
+                let content = "machine example.com login anonymous password qwerty machine example2.com login anonymous2 password qwerty2"
+                
+                let machines = Netrc.from(content).value?.machines
+                expect(machines?.count) == 2
+                
+                var machine = machines?[0]
+                expect(machine?.name) == "example.com"
+                expect(machine?.login) == "anonymous"
+                expect(machine?.password) == "qwerty"
+                
+                machine = machines?[1]
+                expect(machine?.name) == "example2.com"
+                expect(machine?.login) == "anonymous2"
+                expect(machine?.password) == "qwerty2"
+            }
+            
+            it("should load multiple machines for a given multi-line format") {
+                let content = """
+                    machine  example.com login     anonymous
+                    password                  qwerty
+                    machine example2.com
+                    login anonymous2
+                    password qwerty2
+                    """
+                
+                let machines = Netrc.from(content).value?.machines
+                expect(machines?.count) == 2
+                
+                var machine = machines?[0]
+                expect(machine?.name) == "example.com"
+                expect(machine?.login) == "anonymous"
+                expect(machine?.password) == "qwerty"
+                
+                machine = machines?[1]
+                expect(machine?.name) == "example2.com"
+                expect(machine?.login) == "anonymous2"
+                expect(machine?.password) == "qwerty2"
+            }
+            
+            it("should throw error when machine parameter is missing") {
+                let content = "login anonymous password qwerty"
+                let error = Netrc.from(content).error
+                
+                switch error {
+                case .some(.machineNotFound):
+                    break
+                default:
+                    fail("Expected invalidJSON error")
```

Comment:
These fail messages don't make sense to me -- is this parsing JSON?

Reply:
Thanks for spotting that, a copy - paste mistake, fixed ✔️